config/locales/en.yml

```diff
@@ -907,122 +907,122 @@ en:
           winery: "Winery"
           "yes": "Craft Shop"
         emergency:
-          access_point: "Access Point"
+          access_point: "Emergency Access Point"
           ambulance_station: "Ambulance Station"
-          assembly_point: "Assembly Point"
+          assembly_point: "Emergency Assembly Point"
           defibrillator: "Defibrillator"
           fire_extinguisher: "Fire Extinguisher"
           fire_water_pond: "Fire Water Pond"
           landing_site: "Emergency Landing Site"
           life_ring: "Emergency Life Ring"
           phone: "Emergency Phone"
-          siren: "Emergency Siren"
-          suction_point: "Emergency Suction Point"
-          water_tank: "Emergency Water Tank"
+          siren: "Siren"
+          suction_point: "Water Drafting Site"
+          water_tank: "Emergency Water Reservoir"
         highway:
           abandoned: "Abandoned Highway"
-          bridleway: "Bridleway"
-          bus_guideway: "Guided Bus Lane"
+          bridleway: "Bridle Path"
+          bus_guideway: "Guided Busway"
           bus_stop: "Bus Stop"
-          construction: "Highway under Construction"
-          corridor: "Corridor"
+          construction: "Highway Under Construction"
+          corridor: "Indoor Corridor"
           crossing: "Crossing"
-          cycleway: "Cycle Path"
+          cycleway: "Bicycle Path"
           elevator: "Elevator"
           emergency_access_point: "Emergency Access Point"
-          emergency_bay: "Emergency Bay"
+          emergency_bay: "Emergency Stopping Place"
           footway: "Footpath"
           ford: "Ford"
-          give_way: "Give Way Sign"
+          give_way: "Yield Sign"
           living_street: "Living Street"
           milestone: "Milestone"
-          motorway: "Motorway"
-          motorway_junction: "Motorway Junction"
-          motorway_link: "Motorway Road"
+          motorway: "Controlled-Access Highway"
+          motorway_junction: "Highway Exit"
+          motorway_link: "Highway Ramp"
           passing_place: "Passing Place"
           path: "Path"
-          pedestrian: "Pedestrian Way"
-          platform: "Platform"
+          pedestrian: "Pedestrian Mall"
```

Comment:
I've never heard "Pedestrian Mall" in the U.S. Perhaps `Pedestrian Road` would fit better.

Reply:
“Pedestrian mall” has historically been the preferred term in American English, as seen in [Google Books Ngram Viewer](https://books.google.com/ngrams/graph?content=pedestrian+mall%2Cpedestrian+precinct%2Cpedestrian+road%2Cpedestrian+zone&year_start=1920&year_end=2022&corpus=en-US&smoothing=3). However, I appear to be dating myself: “pedestrian street” is more popular in the last several years, and “pedestrian zone” is also becoming common. That’s the term [Wikipedia](https://en.wikipedia.org/wiki/Pedestrian_zone) uses too, to avoid favoring American/Australian terms over the British term “pedestrian precinct”. “Pedestrian road” appears to be very rare.

![A screenshot of Google Books Ngram Viewer showing “pedestrian mall” as by far the preferred term in American English from 1940, peaking in the 1970s, until “pedestrian street” overtook it in 2016. “Pedestrian zone” also ties with it as of 2021. “Pedestrian precinct” and “pedestrian road” are virtually unknown in this dialect.](https://github.com/user-attachments/assets/23866dda-a3a5-45f8-95e6-53433ee945ea)

I think “pedestrian street” would be intuitive and pretty close to your suggestion. “Pedestrian zone” would be problematic because it too strongly suggests an area, whereas this tag can validly appear on a way.

Reply:
Yes, now that you say it I think "Pedestrian Street" is even better. Good idea re: ngram!